File: MySDKs/JAVA SDK/src/main/java/com/example/sdk/SDKException.java
package com.example.sdk;

/**
 * Custom exception for all errors originating from the SDK.
 */
public class SDKException extends Exception {
    public SDKException(String message) {
        super(message);
    }

    public SDKException(String message, Throwable cause) {
        super(message, cause);
    }
}

File: MySDKs/NetCore SDK/SDKException.cs
namespace MyCompany.CryptoSDK;

/// <summary>
/// Custom exception for all errors originating from the SDK.
/// </summary>
public class SDKException : Exception
{
    public SDKException(string message) : base(message) { }
    public SDKException(string message, Exception innerException) : base(message, innerException) { }
}
